### ISSUE TITLE
bug after Update consensus

### ISSUE THREAD

**Issue by tmeinlschmidt** [XinFinOrg/XDPoSChain#377]
Node version
````
INFO [12-07|08:49:08] Starting peer-to-peer node               instance=XDC/v2.0.0-beta1-287ca511/linux-amd64/go1.21.4
````

as we needed that rpc timeout in place for debug

````
INFO [12-07|08:49:43] Signer reward                            signer=xdcE3bC38418f89C386d1093aee89d606564012e329 sign=60 reward=47641734159123392040
INFO [12-07|08:49:43] Signer reward                            signer=xdcB38ABA47A5563fa4AeDC2a649EF819b9FB160b6a sign=60 reward=47641734159123392040
INFO [12-07|08:49:43] Signer reward                            signer=xdcE4710a854b24062fF37Ab6636bd9A456e24c1635 sign=60 reward=47641734159123392040
INFO [12-07|08:49:43] Signer reward                            signer=xdce2BC9A03D5AE35E130C8bF99eA50c022de375DB7 sign=60 reward=47641734159123392040
INFO [12-07|08:49:44] Imported new chain segment               blocks=1   txs=1    mgas=0.000  elapsed=2.263s    mgasps=0.000  number=68873400 hash=4e691c…4ac8d2 cache=0.00B
INFO [12-07|08:49:44] Checkpoint!!! It's time to reconcile node's state...
INFO [12-07|08:49:44] Update consensus parameters
You've encountered a sought after, hard to reproduce bug. Please report this to the developers <3 https://github.com/XinFinOrg/XDPoSChain/issues
index allocation went beyond available resultCache space
/XDPoSChain/eth/downloader/queue.go:510
goroutine 1701 [running]:
runtime/debug.Stack()
	/usr/local/go/src/runtime/debug/stack.go:24 +0x5e
runtime/debug.PrintStack()
	/usr/local/go/src/runtime/debug/stack.go:16 +0x13
github.com/XinFinOrg/XDPoSChain/common.Report({0xc01afeda88, 0x1, 0x1})
	/XDPoSChain/common/debug.go:35 +0x12a
github.com/XinFinOrg/XDPoSChain/eth/downloader.(*queue).reserveHeaders(0xc0022e8f00, 0xc009a54780, 0x80, 0xc009ae0410?, 0xc00023b408, 0x0?, 0xc01afedaf0?, 0x144fe70)
	/XDPoSChain/eth/downloader/queue.go:510 +0x598
github.com/XinFinOrg/XDPoSChain/eth/downloader.(*queue).ReserveBodies(0xd0ffc6?, 0xd27785?, 0xc01afedb48?)
	/XDPoSChain/eth/downloader/queue.go:462 +0xe5
github.com/XinFinOrg/XDPoSChain/eth/downloader.(*Downloader).fetchParts(0xc000535860, {0x15d5de0, 0x1d5a5e0}, 0xc00010f980, 0xc01afedea8, 0xc0000ec7e0, 0xc01afede98, 0xc01afedf08, 0xc01afedef8, 0xc01afedee8, ...)
	/XDPoSChain/eth/downloader/downloader.go:1121 +0xcbb
github.com/XinFinOrg/XDPoSChain/eth/downloader.(*Downloader).fetchBodies(0xc000535860, 0xad7e0d9dc0ca1ad9?)
	/XDPoSChain/eth/downloader/downloader.go:942 +0x29f
github.com/XinFinOrg/XDPoSChain/eth/downloader.(*Downloader).syncWithPeer.func4()
	/XDPoSChain/eth/downloader/downloader.go:471 +0x1e
github.com/XinFinOrg/XDPoSChain/eth/downloader.(*Downloader).spawnSync.func1()
	/XDPoSChain/eth/downloader/downloader.go:491 +0x5d
created by github.com/XinFinOrg/XDPoSChain/eth/downloader.(*Downloader).spawnSync in goroutine 1653
	/XDPoSChain/eth/downloader/downloader.go:491 +0x7a
#### BUG! PLEASE REPORT ####
INFO [12-07|08:49:48] It's time to update new set of masternodes for the next epoch...
INFO [12-07|08:49:48] Ordered list of masternode candidates
INFO [12-07|08:49:48]                                          address=xdc5651290BdD3a952357066B324F9334B544100140 stake=10025000000000000000000000
INFO [12-07|08:49:48]                                          address=xdc90c87C9ff588E9268a1C7c79A5986886E98C2F04 stake=10025000000000000000000000
INFO [12-07|08:49:48]                                          address=xdcB1693d224d4FD70aD6c9C9bDc44faa3ce21Fb40D stake=10025000000000000000000000
````

**Comment by gzliudan** [comment]
If your node can synchronize normally, then this error message is not actually a bug. I have seen many same messages before and after v2 consensus.

**Comment by gzliudan** [comment]
Close this issue because long time no activity. Please feel free to reopen it or create a new one if still has problem.

**Comment by gzliudan** [comment]
Fixed by PR #563.
